packages/clerk-js/src/core/tokenCache.ts

```diff
@@ -27,6 +27,8 @@ interface TokenCache {
 const KEY_PREFIX = 'clerk';
 const DELIMITER = '::';
 const LEEWAY = 10;
+// This value should have the same value as the INTERVAL_IN_MS in AuthenticationPoller
+const SYNC_LEEWAY = 5;
```

Comment:
🙃 In the future we should either import this from the poller, or extracts all these constants in a separate file. We lose the benefits of code colocation, but I think the tradeoff is worth it